src/sources/forms/TaxonomicServiceForm.json

```diff
@@ -6,7 +6,7 @@
             {
                 "jsonPath": "$['schema:service']['schema:serviceType']",
                 "title": "Service Type",
-                "description": "A type that defines the kind of taxonomic service",
+                "description": "A type that defines the kind of taxonomic service, select Service inventory for a website that contains multiple taxonomic services of the same sort, like multiple identification keys",
```

Comment:
`Service` does this need to be capitalized

Reply:
Fixed